app/experimenter/nimbus-ui/src/components/PageEditAudience/index.test.tsx

```diff
@@ -97,6 +97,22 @@ describe("PageEditAudience", () => {
     });
   });
 
+  it("redirects to the review page if the experiment status is preview", async () => {
```

Comment:
Huh, seems weird that we have to test this for each and every page - but I see we're also doing it for Complete status. 

Might be worth a maintenance sprint task to pull all of these into a single test for `editCommonRedirects` in `app/experimenter/nimbus-ui/src/lib/experiment.ts`. If we test anything from page-to-page, it should just be that `editCommonRedirects` is called

Reply:
Deal! https://jira.mozilla.com/browse/EXP-937